python/ray/data/_internal/logical/operators/map_operator.py

```diff
@@ -21,21 +21,17 @@ class AbstractMap(LogicalOperator):
     MapOperator.
     """
 
-    # TODO: Replace `fn`, `fn_args`, `fn_kwargs`, `fn_constructor_args`, and
-    # `fn_constructor_kwargs` from this API, in favor of `block_fn_args` and
-    # `block_fn_kwargs`. Operators should only be concerned with `block_fn`.
     def __init__(
         self,
         name: str,
         input_op: LogicalOperator,
-        block_fn: BlockTransform,
-        compute: Optional[Union[str, ComputeStrategy]] = None,
-        target_block_size: Optional[int] = None,
-        fn: Optional[UDF] = None,
+        fn: UDF,
         fn_args: Optional[Iterable[Any]] = None,
         fn_kwargs: Optional[Dict[str, Any]] = None,
         fn_constructor_args: Optional[Iterable[Any]] = None,
         fn_constructor_kwargs: Optional[Dict[str, Any]] = None,
+        target_block_size: Optional[int] = None,
```

Comment:
This is just reordering arguments, to make all `fn`-related arguments to be put together, and look more coherent.